runtime/gc_vlhgc/SchedulingDelegate.hpp

```diff
@@ -273,8 +273,9 @@ class MM_SchedulingDelegate : public MM_BaseNonVirtual
 	 * This is typically the same as GCExtensions->tarokeEdenSize, but may be smaller if 
 	 * insufficient memory is available
 	 * @param env[in] the main GC thread
+	 * @param allowTotalHeapResize[in] counting maxHeapExpansionRegions for adjuesting eden size,default is false
```

Comment:
Space after comma, and please correct the spelling of "adjusting".
I think "count" would be better than than "counting" here.
Please remove the trailing space on line 273 and add the missing period on line 274.